File: countDigit.java
import java.util.*;
class CountDigit{
public static void main(String[] args){
        int a=1291;
        int ans=countDigit(a);
        System.out.println(ans);
}
public static int countDigit(int a){
    int n=a;
    int count=0;
    while(n!=0){
       
        count++;
        n=n/10;
    }
    return count;
} 

}

File: countDigit.js
// function countDigit(a){
//     let cnt=0;
//     while(a!==0){
//         cnt++;
//         a=Math.floor(a/10);
//     }
//     return cnt;

// }
// let a=12121212
// let ans=countDigit(a);
// console.log(ans)

const countDigit =(n)=> {
    let cnt=0;
    while(n>0){
        cnt++;
        n=Math.floor(n/10);
    }
    return cnt;

}
let n=1212898945;

console.log(countDigit(n));

